src/ensembl/src/content/app/entity-viewer/gene-view/components/default-transcripts-list/default-transcripts-list-item/DefaultTranscriptListItem.tsx

```diff
@@ -33,6 +33,7 @@ import styles from './DefaultTranscriptListItem.scss';
 
 export type DefaultTranscriptListItemProps = {
   gene: Gene;
+  label: string;
```

Comment:
Wouldn't `isDefault: boolean` be a better choice?

Reply:
Label can be Canonical/Mane Select etc. coming from Thoas

Reply:
If so, won't you be able to get the actual label from the transcript that is being passed to the TranscriptListItem component?

If a TranscriptListItem component knows that it is the default one, it can look into the transcript object and find the relevant field to show the label.

Reply:
yes thats the end goal. but can we add in fields to the transcript object atm?

Reply:
> yes thats the end goal

There are two ways I can see for showing the label:

1) Something outside the `DefaultTranscriptListItem` component decides why this particular item is the default one and what text it is going to show. This something will also have to decide what to show in the question button here.

![image](https://user-images.githubusercontent.com/6834224/91048600-cd382f80-e613-11ea-8890-d49ef9eb6292.png)

2) Alternatively, the `DefaultTranscriptListItem` component has only to be notified from the outside that it happens to be the default component, and then it will itself decide what text to show in the label, and what text to show in the question button.

Option 2 looks simpler at the moment; so I suggest we pass the `isDefault: boolean` prop to the `DefaultTranscriptListItem`, which will show the hard-coded `"Selected"` string as a label if it is a default transcript.

Would love to hear what @msmanoj or @imransl think.

Reply:
Can there be a Selected and Canonical in the list of transcripts for a gene?

Reply:
According to the [CDM](https://www.ebi.ac.uk/seqdb/confluence/display/ENS/Mini+hackathon+-+2020-08-21) (super rough draft, nothing certain), each transcript will have a `canonical: boolean` field; and one of the transcripts of a gene might also have a `mane: 'select'` field. Mane is available only for several species though.

Reply:
'Selected' is just a placeholder until we have real data. When we do then each gene will have one transcript that is 'Ensembl canonical', which as Andrey mentions will always be the 'MANE select' if there is one (human 38 only).

In 102 there are genes that have both a 'MANE Select' and a 'MANE Plus' transcript. I think we should be showing both of these on the transcript image. Do we need clarification of that @jyothishnt ?

Reply:
I think we need to code to show one or more titles based on what comes from the payload. Could be MANE Select or MANE Plus, or anything that comes in the future.